tests/release/pipelines/e2e-fbc.go

```diff
@@ -0,0 +1,166 @@
+			}},
+			Configuration: &ecp.EnterpriseContractPolicyConfiguration{
+				Collections: []string{"minimal"},
+				Exclude:     []string{"cve", "step_image_registries"},
+				Include:     []string{"@slsa1", "@slsa2", "@slsa3"},
+			},
+		}
+
+		_, err = managed_fw.AsKubeDeveloper.TektonController.CreateEnterpriseContractPolicy(fbcEnterpriseContractPolicyName, managedNamespace, defaultEcPolicySpec)
+		Expect(err).NotTo(HaveOccurred())
+
+	})
+
+	AfterAll(func() {
+		if !CurrentSpecReport().Failed() {
+			Expect(dev_fw.AsKubeDeveloper.HasController.DeleteApplication(fbcApplicationName, devNamespace, false)).NotTo(HaveOccurred())
+			Expect(managed_fw.AsKubeDeveloper.TektonController.DeleteEnterpriseContractPolicy(fbcEnterpriseContractPolicyName, managedNamespace, false)).NotTo(HaveOccurred())
+			Expect(managed_fw.AsKubeDeveloper.ReleaseController.DeleteReleaseStrategy(fbcReleaseStrategyName, managedNamespace, false)).NotTo(HaveOccurred())
+			Expect(managed_fw.AsKubeDeveloper.ReleaseController.DeleteReleasePlanAdmission(fbcReleasePlanAdmissionName, managedNamespace, false)).NotTo(HaveOccurred())
+		}
+	})
+
+	var _ = Describe("Post-release verification", func() {
+
+		It("verifies that a build PipelineRun is created in dev namespace and succeeds", func() {
+			Expect(dev_fw.AsKubeDeveloper.HasController.WaitForComponentPipelineToBeFinished(component, "", 2, dev_fw.AsKubeDeveloper.TektonController)).To(Succeed())
+			buildPr, err = dev_fw.AsKubeDeveloper.HasController.GetComponentPipelineRun(component.Name, fbcApplicationName, devNamespace, "")
+			Expect(err).ShouldNot(HaveOccurred())
+			snapshot, err = dev_fw.AsKubeDeveloper.IntegrationController.GetSnapshot("", buildPr.Name, component.Name, devNamespace)
+			Expect(err).ShouldNot(HaveOccurred())
+			releaseCR, err = dev_fw.AsKubeDeveloper.ReleaseController.GetRelease("", snapshot.Name, devNamespace)
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+
+		It("verifies the fbc release pipelinerun is running and succeeds", func() {
+			Eventually(func() error {
+				releasePr, err = managed_fw.AsKubeAdmin.ReleaseController.GetPipelineRunInNamespace(managed_fw.UserNamespace, releaseCR.GetName(), releaseCR.GetNamespace())
+				Expect(err).ShouldNot(HaveOccurred())
+				if !releasePr.IsDone() {
+					return fmt.Errorf("release pipelinerun %s in namespace %s did not finish yet", releasePr.Name, releasePr.Namespace)
+				}
+				Expect(utils.HasPipelineRunSucceeded(releasePr)).To(BeTrue(), fmt.Sprintf("release pipelinerun %s/%s did not succeed", releasePr.GetNamespace(), releasePr.GetName()))
+				return nil
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(Succeed(), "timed out when waiting for release pipelinerun to succeed")
+		})
+
+		It("verifies release CR completed and set succeeded.", func() {
+			Eventually(func() error {
+				releaseCR, err = dev_fw.AsKubeDeveloper.ReleaseController.GetFirstReleaseInNamespace(dev_fw.UserNamespace)
```

Comment:
what if there are more Releases in the namespace? should we rather get a Release based on the Snapshot name?